docs/advanced-usage/installation.md

```diff
@@ -66,11 +66,16 @@ yay -S spicetify-cli
 
 Before applying Spicetify, you need to gain write permission on Spotify files, by running command:
 
+<!--This may be outdated as the AUR package for the spotify launcher uses a different install directory-->
```

Comment:
It looks like `spotify-launcher` is in the [extra] repo, not AUR. Or are you referring to something else?

Reply:
Oh, sorry. I was referring to spotify-launcher, must have written the wrong thing. Sorry about that, it is in [extra]

Reply:
Then this comment does not really apply right? Since it does still apply, just for a different installation method.
```suggestion
```